lms/validation/authentication/_bearer_token.py

```diff
@@ -109,69 +84,32 @@ def lti_user(self, location):
           ``"Bearer <ENCODED_JWT>"`` format.
         :raise ValidationError: if the JWT's payload is invalid, for example if
           it's missing a required parameter
-
-        :rtype: LTIUser
         """
-        try:
-            return self.parse(location=location)
-        except ValidationError as error:
-            try:
-                authorization_error_message = error.messages[location]["authorization"][
-                    0
-                ]
-            except KeyError:
```

Comment:
Lots of guessing what exception to raise, but the right type is just below in a different method.